scripts/plant_life_hist_eu.py

```diff
@@ -68,7 +68,7 @@ def download(self, engine=None, debug=False):
             new_data.close()
             self.engine.auto_create_table(Table(key,
                                                 cleanup=Cleanup(correct_invalid_value,
-                                                                nulls=[-999.9])),filename=str("new" + key))
+                                                                missingValues=[-999.9])),filename=str("new" + key))
```

Comment:
@ethanwhite I used `retriever ls` to test the scripts. It worked fine, but I a too had this doubt about missingValues in .py scripts, just to be clear are you referring to something similar to this